distribution/docs/src/main/resources/_contents/_configuring/configuring-from-configuration-files-contents.adoc

```diff
@@ -0,0 +1,746 @@
+
+|Lists of Integers
+|`name = I[ "value1", "value1", ... ]`
+|`sizes = I[ "10", "20", "30" ]`
+
+|===
+
+[NOTE]
+====
+* Lists of values can be prefixed with any of the supported types (`B`, `I`, `L`, `F` or `D`)
+* To prevent any configuration issues, the `=` signs used in values should be escaped using a backslash (`\`)
+* Boolean values will default to `false` if any value other than `true` is provided
+* Escape character in values must be used for double quotes (`"`) and spaces, but cannot be used with `{` `}` or `[` `]` pairings
+====
+
+.Sample configuration file
+[source,java,linenums]
+----
+service.pid="org.codice.ddf.security.policy.context.impl.PolicyManager"
+
+authenticationTypes=["/\=SAML|GUEST","/admin\=SAML|basic","/system\=basic","/solr\=SAML|PKI|basic","/sources\=SAML|basic","/security-config\=SAML|basic","/search\=basic"]
+
+realms=["/\=karaf"]
+
+requiredAttributes=["/\=","/admin\={http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role\=admin}","/solr\={http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role\=admin}","/system\={http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role\=admin}","/security-config\={http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role\=admin}"]
+
+whiteListContexts=["/services/SecurityTokenService","/services/internal/metrics","/services/saml","/proxy","/services/csw"]
+----
+
+===== Editing HTTP Ports for Multiple Local ${branding} Nodes
+
+Edit the port numbers in the files in the ${branding} install folder.
+
+[cols="4" options="header"]
+|===
+
+|File to Edit
+|Property(ies)
+|Original Value
+|Example of New Value
+
+|`bin/karaf.bat`
+|`address`
+|`5005`
+|`5006`
+
+.2+|`etc/org.apache.karaf.management.cfg`
+|`rmiRegistryPort`
+|`1099`
+|`1199`
+
+
+|`rmiServerPort`
+|`44444`
+|`44445`
+
+.2+|`etc/system.properties`
+|`httpsPort`,`port`
+|`8993`
+|`8994`
+
+|`httpPort`
+|`8181`
+|`8281`
+
+|===
+
+==== Editing ${branding} Web Service Providers Configuration Files
+
+[IMPORTANT]
+====
+If the hostname is changed during the install to something other than `localhost` a new keystore and truststore must be provided.
+====
+
+[TIP]
+====
+When changing the hostname for testing or development purposes, the installer can be started with a `?dev=true` URL query parameter. This causes the system to automatically generate self-signed certificates for any hostname that is entered during the install process.
+====
+
+==== Configuring Files in HOME Directory Hierarchy
+
+Many important configuration settings are stored in the `<${branding}_HOME>` directory.
+
+[IMPORTANT]
+====
+The passwords configured in this section reflect the passwords used to decrypt JKS (Java KeyStore) files.
+Changing these values without also changing the passwords of the JKS causes undesirable behavior.
+====
+
+. In `<${branding}_HOME>/etc/users.properties`, modify the line: `localhost=localhost,group,admin,manager,viewer,system-admin` to be: `<FQDN>=<PASSWORD>,group,admin,manager,viewer,system-admin`
```

Comment:
Not needed. Concepts already covered in section `Updating System Users` in section 4.1.4.